sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items-mlflow.ipynb

```diff
@@ -0,0 +1,1194 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1634852261744
+    },
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "name": "mlclient-setup",
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "credential = DefaultAzureCredential()\n",
+    "ml_client = None\n",
+    "try:\n",
+    "    ml_client = MLClient.from_config(credential)\n",
+    "except Exception as ex:\n",
+    "    print(ex)\n",
+    "    # Enter details of your AML workspace\n",
+    "    subscription_id = \"<SUBSCRIPTION_ID>\"\n",
+    "    resource_group = \"<RESOURCE_GROUP>\"\n",
+    "    workspace = \"<AML_WORKSPACE_NAME>\"\n",
+    "    ml_client = MLClient(credential, subscription_id, resource_group, workspace)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. MLTable with input Training Data\n",
+    "\n",
+    "In order to generate models for computer vision tasks with automated machine learning, you need to bring labeled image data as input for model training in the form of an MLTable. You can create an MLTable from labeled training data in JSONL format. If your labeled training data is in a different format (like, pascal VOC or COCO), you can use a conversion script to first convert it to JSONL, and then create an MLTable. Alternatively, you can use Azure Machine Learning's [data labeling tool](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-create-image-labeling-projects) to manually label images, and export the labeled data to use for training your AutoML model.\n",
+    "\n",
+    "In this notebook, we use a toy dataset called Fridge Objects, which consists of 134 images of 4 classes of beverage container {can, carton, milk bottle, water bottle} photos taken on different backgrounds.\n",
+    "\n",
+    "All images in this notebook are hosted in [this repository](https://github.com/microsoft/computervision-recipes) and are made available under the [MIT license](https://github.com/microsoft/computervision-recipes/blob/master/LICENSE).\n",
+    "\n",
+    "**NOTE:** In this PRIVATE PREVIEW we're defining the MLTable in a separate folder and .YAML file.\n",
+    "In later versions, you'll be able to do it all in Python APIs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.1. Download the Data\n",
+    "We first download and unzip the data locally."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import urllib\n",
+    "from zipfile import ZipFile\n",
+    "\n",
+    "# download data\n",
+    "download_url = \"https://cvbp-secondary.z19.web.core.windows.net/datasets/image_classification/fridgeObjects.zip\"\n",
+    "data_file = \"./data/fridgeObjects.zip\"\n",
```

Comment:
As we discussed, we assume that data directory is present in current directory. otherwise, following error arises.
checking for the directory and if it is not present then creating one helps.
`FileNotFoundError                         Traceback (most recent call last)
~\AppData\Local\Temp\ipykernel_16680\908443178.py in <module>
      6 download_url = "https://cvbp-secondary.z19.web.core.windows.net/datasets/image_classification/multilabelFridgeObjects.zip"
      7 data_file = "./data/multilabelFridgeObjects.zip"
----> 8 urllib.request.urlretrieve(download_url, filename=data_file)
      9 
     10 # extract files

~\Anaconda3\envs\xaibenchmarks\lib\urllib\request.py in urlretrieve(url, filename, reporthook, data)
    255         # Handle temporary file setup.
    256         if filename:
--> 257             tfp = open(filename, 'wb')
    258         else:
    259             tfp = tempfile.NamedTemporaryFile(delete=False)

FileNotFoundError: [Errno 2] No such file or directory: './data/multilabelFridgeObjects.zip'
`

Reply:
@PhaniShekhar I think checking for the data directory makes sense. should i add it to all the 8 dpv2 notebooks?

Reply:
We need the data directory to be present as well since we need the MLTable files which are present in those folders. Maybe we can add to call to create the data directory if not present. Something like
os.makedirs("./data", exist_ok=True)